src/v/cluster/tests/tx_compaction_tests.cc

```diff
@@ -54,6 +56,9 @@ FIXTURE_TEST(test_tx_compaction_combinations, mux_state_machine_fixture) {
     // batches and tx control batches removed.
     // Each workload execution can fully be backtracked from the test log
     // (in case of failures) and re-executed manually.
+    config::shard_local_cfg().log_disable_housekeeping_for_tests.set_value(
+      true);
+    config::shard_local_cfg().log_segment_ms_min.set_value(1ms);
```

Comment:
You should either reset these manually at the end of the test or use a scoped_config to set these, otherwise they'll bleed into other tests

Reply:
oo, good point, fixed